pkg/kudoctl/cmd/init.go

```diff
@@ -168,10 +175,6 @@ func (initCmd *initCmd) run() error {
 		}
 
 		if err := cmdInit.Install(initCmd.client, opts, initCmd.crdOnly); err != nil {
-			if apierrors.IsAlreadyExists(err) {
```

Comment:
this removed message is still necessary IMO... unless we want to change line 108 of manager.go.
Most of the installs `return nil` if `isAlreadyExistsError` returns true... with the exception of the last install which is the kudo controller itself.   If the kudo controller exists then it returns the error and this warning message was printed.

For clarity:   this is what most install services do: https://github.com/kudobuilder/kudo/blob/528bab07e1f4a109a8f2c08206e78f9d44a21139/pkg/kudoctl/cmd/init/manager.go#L98

This is what install the service does:  https://github.com/kudobuilder/kudo/blob/528bab07e1f4a109a8f2c08206e78f9d44a21139/pkg/kudoctl/cmd/init/manager.go#L106-L111

I'm marking as reject because we either:
1) need to return nil from the install of the statefulset (along with the removal of this warning message) or
2) need to return warning message.

Reply:
Interesting, I wonder what to do here. I don't really have any immediate answer, have to think about it...

but that means you cannot run init twice because the second one will always fail because the manager is already running, right?

I am actually starting to think that the correct behavior of init here is to do an update when the thing already exists. Because manager is not the only thing that if it exists, it means it is in the same state as what we're trying to apply 🤔 Let me know what you think... I think that way, it would be truly idempotent which would be lovely...